test/integration/arbitrum-mainnet/Loans.fork.t.sol

```diff
@@ -549,7 +562,18 @@ contract LoansForkTest is LoansTestBase {
 
         // Verify balances
         assertEq(pair.underlying.balanceOf(user), userUnderlyingBefore - underlyingAmount - expectedEscrowFee);
-        assertEq(pair.underlying.balanceOf(escrowSupplier), escrowSupplierUnderlyingBefore);
+        assertEq(escrowSupplierUnderlyingBefore - pair.underlying.balanceOf(escrowSupplier), underlyingAmount);
+    }
+
+    function checkLoanAmount(uint actualLoanAmount) internal view {
+        uint oraclePrice = pair.takerNFT.currentOraclePrice();
+        uint expectedCashFromSwap = underlyingAmount * oraclePrice / pair.oracle.baseUnitAmount();
+        // Calculate minimum expected loan amount (expectedCash * LTV)
+        // Apply slippage tolerance for swaps and rounding
+        uint minExpectedLoan = expectedCashFromSwap * pair.ltvs[0] * (100 - slippage) / (BIPS_BASE * 100);
+
+        // Check actual loan amount is at least the minimum expected
+        assertGe(actualLoanAmount, minExpectedLoan);
```

Comment:
btw, there's a variant of assertEq that accepts tolerance. Might be more suitable here: maybe more readable, and checks both directions < >.